recipes/open-webui/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "open-webui",
+  "name": "Open WebUI",
+  "version": "1.0.0",
+  "license": "MIT",
+  "config": {
+    "hasNotificationSound": true,
+    "hasHostedOption": false,
```

Comment:
This `hasHostedOption` config can be removed since it is set to false by default.

Reply:
Fixed